functests/utils/pods/pods.go

```diff
@@ -143,3 +145,23 @@ func GetContainerIDByName(pod *corev1.Pod, containerName string) (string, error)
 	}
 	return "", fmt.Errorf("failed to find the container ID for the container %q under the pod %q", containerName, pod.Name)
 }
+
+// GetPerformanceProfilePod returns the pod running the Performance Profile Operator
+func GetPerformanceProfilePod() (*corev1.Pod, error)  {
+	selector, err := labels.Parse(fmt.Sprintf("%s=%s", "name", "performance-operator"))
+	if err != nil {
+		return nil, err
+	}
+
+	pods := &corev1.PodList{}
+
+	opts := &client.ListOptions{LabelSelector: selector, Namespace: testutils.PerformanceOperatorNamespace}
+	if err := testclient.Client.List(context.TODO(), pods, opts); err != nil {
+		return nil, err
+	}
+	if len(pods.Items) != 1 {
+		return nil, fmt.Errorf("incorrect performance operator pods count: %d", len(pods.Items))
+	}
+
```

Comment:
any way to check that pod is indeeded running `performance-operator` and not any random pod which is happen to have the same label?

Reply:
In the same namespace as PAO  "openshift-performance-addon" ? I think is a relatively safe bet...

Reply:
It's still a bet, meaning is still anecdotal evidence, and we need quite some of it to gain confidence. Checking the image name seems stronger (does it contain 'performance-operator'). Feel free to add more checks, or to lookup better ones.
The best way would probably be to check a PAO endpoint and to verify it is behaving correctly; that's sufficient proof the service is there. It's probably complex, maybe too complex to be done here, though.

Reply:
We are also checking we have only one pod in the namespace. The check is the first step in the func tests, I suppose the next steps will fail if for some reason the PAO is not in the namespace, but another pod with same label.
While I agree that theoretically is not enough, is it the place to add more checks?
I can add a check the actual pod name starts with performace-operator-{something}, but this is a side effect on how k8s names pods.Or maybe would add to confidence?

Reply:
Added check the performance operator pod name (not label) starts with "performance-operator"

Reply:
That should increase the confidence in the bet

Reply:
Still not completely happy with this solution, but good enough for now.